listenbrainz/webserver/views/index.py

```diff
@@ -103,11 +106,26 @@ def current_status():
     except DatabaseException as e:
         user_count = 'Unknown'
 
+    listen_counts_per_day: List[dict] = []
+    for delta in range(2):
```

Comment:
constant in code?

Reply:
I'd say let's keep it like this for now, we'll have to change some stuff anyways if we want to show more data.